<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsPointCloudAttributeByRampRendererWidgetBase</class>
 <widget class="QWidget" name="QgsPointCloudAttributeByRampRendererWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>781</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,1">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0,1,0">
     <item>
      <widget class="QLabel" name="mScalarMinLabel">
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QgsDoubleSpinBox" name="mMinSpin">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-9999999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mScalarMaxLabel">
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QgsDoubleSpinBox" name="mMaxSpin">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-9999999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mScalarRecalculateMinMaxButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Attribute</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QgsPointCloudAttributeComboBox" name="mAttributeComboBox"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QgsColorRampShaderWidget" name="mScalarColorRampShaderWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsPointCloudAttributeComboBox</class>
   <extends>QComboBox</extends>
   <header>qgspointcloudattributecombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsColorRampShaderWidget</class>
   <extends>QWidget</extends>
   <header>qgscolorrampshaderwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
